index.bs

```diff
@@ -1081,6 +1115,104 @@ The <dfn attribute for="XRInputSource">targetRaySpace</dfn> attribute is an {{XR
+
+ISSUE: Document {{Gamepad/id}} restrictions and formatting.
+</section>
+
+<section class="unstable">
+"xr-standard" Gamepad Mapping {#xr-standard-gamepad-mapping}
+-----------------------------
+
+The WebXR Device API introduces a new {{GamepadMappingType}} to describe the mapping of common XR controller devices.
+
+<pre class="idl">
+  enum GamepadMappingType {
+    "xr-standard",
+  };
+</pre>
+
+The <dfn enum-value for="GamepadMappingType">xr-standard</dfn> mapping indicates that the layout of the buttons and axes of the {{XRInputSource/gamepad}} corresponds as closely as possible to the tables below. In order to report a {{Gamepad/mapping}} of {{GamepadMappingType/xr-standard}} the device MUST report a {{XRInputSource/targetRayMode}} of {{XRTargetRayMode/tracked-pointer}}, MUST have at least one touchpad or joystick and MUST have at least one primary button, often a trigger, separate from the touchpad or joystick. If a device does not meet the requirements for the {{GamepadMappingType/xr-standard}} mapping it may still be exposed on an {{XRInputSource}} with the <code>""</code> (empty string) mapping.
+
+<table class="tg">
+  <thead>
+    <tr>
+      <th>Button</th>
+      <th><code>xr-standard</code> Location</th>
+      <th>Required</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>buttons[0]</td>
+      <td>Primary Button/Trigger</td>
+      <td>Yes</td>
+    </tr>
+    <tr>
+      <td>buttons[1]</td>
+      <td>Primary Touchpad/Joystick Button</td>
+      <td>Yes</td>
+    </tr>
+    <tr>
+      <td>buttons[2]</td>
+      <td>Secondary Button/Grip Trigger</td>
+      <td>No</td>
+    </tr>
+    <tr>
+      <td>buttons[3]</td>
+      <td>Secondary Touchpad/Joystick Button</td>
+      <td>No</td>
+    </tr>
+  </tbody>
+</table>
+<br/>
+<table class="tg"s>
+  <thead>
+    <tr>
+      <th>Axis</th>
+      <th><code>xr-standard</code> Location</th>
+      <th>Required</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>axis[0]</td>
+      <td>Primary Touchpad/Joystick X</td>
+      <td>Yes</td>
+    </tr>
+    <tr>
+      <td>axis[1]</td>
+      <td>Primary Touchpad/Joystick Y</td>
+      <td>Yes</td>
+    </tr>
+    <tr>
+      <td>axis[2]</td>
+      <td>Secondary Touchpad/Joystick X</td>
+      <td>No</td>
+    </tr>
+    <tr>
+      <td>axis[3]</td>
+      <td>Secondary Touchpad/Joystick Y</td>
+      <td>No</td>
+    </tr>
+  </tbody>
+</table>
+
+Additional buttons or axes may be exposed after these reserved indices, and SHOULD appear in order of decreasing importance. Devices that lack one of the inputs listed in the tables above MUST still preserve their place in the {{Gamepad/buttons}} or {{Gamepad/axes}} array. If a device has both a touchpad and a joystick the UA MAY designate whichever it chooses to be the primary axis-based input. Buttons reserved by the UA or platform MUST NOT be exposed.
```

Comment:
Can we call out that additional X/Y axis things should always report the X axis before the Y?